### PR TITLE
arm64 support

### PR DESCRIPTION
**Pull request by jontg**
Ref #11

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,10 +29,12 @@ run-backend:
 dist: dist-frontend dist-backend
 dist-frontend:
 	yarn build
-dist-backend: dist-backend-linux_amd64 dist-backend-linux_arm dist-backend-darwin_amd64 dist-backend-windows_amd64
+dist-backend: dist-backend-linux_amd64 dist-backend-linux_arm dist-backend-linux_arm64 dist-backend-darwin_amd64 dist-backend-windows_amd64
 dist-backend-windows_amd64: extension = .exe
 dist-backend-linux_arm:
 	env GOOS=linux GOARCH=arm GOARM=6 go build -ldflags="-s -w" -mod=vendor -o ./dist/strava-plugin_linux_arm ./pkg
+dist-backend-linux_arm64:
+	env GOOS=linux GOARCH=arm64 GOARM=6 go build -ldflags="-s -w" -mod=vendor -o ./dist/strava-plugin_linux_arm64 ./pkg
```
Comment (alexanderzobnin) [inline Makefile:37]: It looks like `GOARM` is not applicable to the `GOARCH=arm64`:
https://github.com/golang/go/wiki/GoArm#supported-architectures

**Review by alexanderzobnin — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a minor change required.

File: Makefile (hunk 1 above)
Comment (jontg) [reply Makefile:37]: Super weird - I should be able to address this tonight, but the compiled binary appears to work on my pi cluster :-D

Comment (alexanderzobnin) [inline Makefile:37]:
```suggestion
	env GOOS=linux GOARCH=arm64 go build -ldflags="-s -w" -mod=vendor -o ./dist/strava-plugin_linux_arm64 ./pkg
```

**Review by alexanderzobnin — CHANGES_REQUESTED** [review]

**Review by alexanderzobnin — APPROVED** [review]

**Pull request merged**